base/terminfo.jl

```diff
@@ -245,7 +245,8 @@ end
 Locate the terminfo file for `term`, return `nothing` if none could be found.
 
 The lookup policy is described in `terminfo(5)` "Fetching Compiled
-Descriptions".
+Descriptions". A terminfo database is included by default with Julia and is
+taken to be the first entry of `@TERMINFO_DIRS@`.
```

Comment:
While `@TERMINFO_DIRS@` came up while discussing the manpage (and was helpful to distinguish from the environment variable of the same name), I don't think this should actually be included in the docstring like this since it's a template pattern that's replaced when documentation is built: people who run `man terminfo(5)` on their system won't see `@TERMINFO_DIRS@` mentioned :sweat_smile:.

Reply:
I actually do see it mentioned on macOS:

> - Finally, ncurses searches these compiled-in locations:
>     - a list of directories (@TERMINFO_DIRS@), and
>     - the system terminfo directory, /usr/share/terminfo (the compiled-in default).

Reply:
I think that's just a broken docs build :stuck_out_tongue: the source text is:

       •   Finally, ncurses searches these compiled-in locations:
           •   a list of directories (@TERMINFO_DIRS@), and
           •   the system terminfo directory, @TERMINFO@ (the compiled-
               in default).

In your example, only `@TERMINFO@` is substituted.

On my system, for comparison:

         •   Finally, ncurses searches these compiled‐in locations:
             •   a list of directories (/etc/terminfo:/usr/share/terminfo), and
             •   the system terminfo directory, /usr/share/terminfo

Reply:
Ha, incredible. Well, the function isn't really user-facing anyway; I'm sure a sufficiently motivated individual could decipher what this docstring means. 😅